Std/Data/Array/Lemmas.lean

```diff
@@ -5,7 +5,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Gabriel Ebner
 -/
 import Std.Data.List.Lemmas
-import Std.Data.Array.Init.Lemmas
```

Comment:
Maybe these files should be left in place but empty? It's annoying to have to recreate them and the header stuff every time.

Reply:
I'd prefer to hope to not need to recreate them? They don't seem inevitable!

Reply:
I've now added them.